### ISSUE TITLE
RPM Relocation broken because of duplication prefix in rpm

### ISSUE THREAD

**Issue by zunkree** [nebula-plugins/gradle-ospackage-plugin#174]
In my build.gradle i have only one call of prefix for rpm package:

```
prefix '/opt'
```

but in resulted rpm it's duplicated

```
$ rpm -qip packagename.rpm | grep 'Relocations:'

Relocations: /opt /opt
```

**Comment by zunkree** [comment]
https://github.com/nebula-plugins/gradle-ospackage-plugin/pull/175

**Comment by imavroukakis** [comment]
Well interesting. If I specify the `prefix` within `buildRpm` like so

```
buildRpm {
    prefix "${mxBaseDir}/apps/${project.name}"
}
```
there's no duplication

**Comment by sghill** [comment]
Closing as #175 has now been merged and released in v4.2.0
